build-logic/build.gradle.kts

```diff
@@ -1,5 +1,7 @@
+import org.jetbrains.kotlin.gradle.dsl.KotlinVersion
+
 plugins {
-    `kotlin-dsl`
+    id("org.gradle.kotlin.kotlin-dsl") version "4.4.0"
```

Comment:
I'm assuming this is for Kotlin 2.0 support right?

Reply:
No, this is just a greater version of the plugin, but 2.0.0 isn't provided yet, this is leftover code from testing it